content-scripts/src/modules/login.js

```diff
@@ -30,26 +31,40 @@ export const rememberLogin = async (data) => {
         //check if there is a error while loading page
         const htmlRes = document.createElement('html');
         htmlRes.innerHTML = await res.text();
```

Comment:
Maybe we should use the `fetchSigarraPage` function here, I remember there being some hiccups with encodings when I did it like this